.github/workflows/ci.yml

```diff
@@ -529,6 +529,8 @@ jobs:
             - { modules: plugin/trino-hive, profile: test-parquet }
             - { modules: plugin/trino-hudi }
             - { modules: plugin/trino-iceberg }
+            - { modules: plugin/trino-iceberg, profile: jdbc-tests }
+            - { modules: plugin/trino-iceberg, profile: rest-tests }
```

Comment:
Would it be OK to run JDBC and REST catalogs together (in one profile for simplicity, or in two profiles, but one CI task)?
Or, is each of them too big to have them together?

Reply:
I suppose the single job is okay. Merged into one.